libraries/abstractions/secure_sockets/lwip/iot_secure_sockets.c

```diff
@@ -943,7 +943,14 @@ static void lwip_dns_found_callback( const char * name,
 {
     uint32_t * addr = ( uint32_t * ) callback_arg;
 
-    *addr = *( ( uint32_t * ) ipaddr ); /* NOTE: IPv4 addresses only */
+    if( ipaddr != NULL )
```

Comment:
Can a test case be added for this?

Reply:
Yes, but currently there are no tests where an additional one can be added. We are improving the test infrastructure and the coverage across the code base.